src/main/play-doc/operation/DynamicProxyConfiguration.md

```diff
@@ -180,6 +209,260 @@ In the example above, the files required are placed within the `/tmp` directory.
+
+> The steps are the same as in non-sandbox mode. However, the configuration values differ from the non-sandbox environment since HAProxy is running as a docker container on the Sandbox.
+
+We will go through the building bundle configuration for HAProxy that can be loaded into the Sandbox. This will allow you to modify the bundle configuration as you see fit.
+
+### Setup
+
+Create a directory where the custom HAProxy configuration will be placed, e.g.
+
+```lang-none
+mkdir -p /tmp/custom-haproxy-conf
+```
+
+### Declare the custom HAProxy configuration entry point
+
+Create a file called `runtime-config.sh` within the proxy configuration directory, e.g.
+
+```lang-none
+touch /tmp/custom-haproxy-conf/runtime-config.sh
+```
+
+Populate the file with the following entry:
+
+```lang-none
+#!/bin/bash
+
+export CONDUCTR_HAPROXY_USE_SUDO="off"
+
+# HAProxy instance will be running inside Docker container, hence access will be controlled via docker run -p options.
+export HAPROXY_BIND_HOST=0.0.0.0
+
+CONFIG_DIR=$( cd $( dirname "${BASH_SOURCE[0]}" ) && pwd )
+export CONDUCTR_HAPROXY_CONFIG_OVERRIDE="$CONFIG_DIR/haproxy-override.cfg"
+export HAPROXY_RELOAD_SCRIPT_SOURCE="$CONFIG_DIR/reloadHAProxy.sh"
+
+SANDBOX_PROXY_DIR="${HOME}/.conductr/proxy"
+SANDBOX_HAPROXY_CFG_DIR="${SANDBOX_PROXY_DIR}/haproxy"
+
+export HAPROXY_RELOAD_SCRIPT_LOCATION="${SANDBOX_PROXY_DIR}/reloadHAProxy.sh"
+export HAPROXY_CONFIG_FILE="${SANDBOX_HAPROXY_CFG_DIR}/haproxy.cfg"
+
+# Ensure directory exists
+mkdir -p "${SANDBOX_HAPROXY_CFG_DIR}"
+
+# Ensure empty reload script exists with correct permission so it can be copied over
+touch "${HAPROXY_RELOAD_SCRIPT_LOCATION}"
+chmod u+x "${HAPROXY_RELOAD_SCRIPT_LOCATION}"
+```
+
+`CONDUCTR_HAPROXY_USE_SUDO` is set to `off` since the HAProxy will be running as a docker container, as such the reload script needs to be run as the same user running the sandbox (hence no `sudo`).
+
+`HAPROXY_BIND_HOST` is set to `0.0.0.0` since the actual HAProxy is running within the docker container, and there's no way to tell what the actual bind ip will be until the container is started. The restriction of the incoming traffic to the HAProxy will be provided by the ports being opened into the container.
+
+The `HAPROXY_RELOAD_SCRIPT_LOCATION` is located at `~/.conductr/proxy/reloadHAProxy.sh`.
+
+The `HAPROXY_CONFIG_FILE` is located at `~/.conductr/proxy/haproxy/haproxy.cfg`. The directory `~/.conductr/proxy/haproxy` is mounted into the container so the `haproxy.cfg` can be read by the HAProxy running within the container.
+
+### Create HAProxy reload script
+
+Create the custom HAProxy reload script, e.g.
+
+```lang-none
+touch /tmp/custom-haproxy-conf/reloadHAProxy.sh
+```
+
+Populate the file with the following template.
+
+```lang-none
+#!/bin/bash
+
+set -e
+docker kill -s HUP sandbox-haproxy
```

Comment:
Possibly explain that this results in the docker container restarting. (probably because the container is setup to always restart when it is killed?)